docs/iris/example_code/Meteorology/COP_1d_plot.py

```diff
@@ -81,8 +81,7 @@ def main():
                 label='pre-industrial', lw=1.5)
 
     # Establish where r and t have the same data, i.e. the observations
-    common = np.where(a1b_mean.data == e1_mean.data)[0]
-    observed = a1b_mean[common]
+    observed = a1b_mean[:np.argmin(np.isclose(a1b_mean.data, e1_mean.data))]
```

Comment:
> In part because it does so much in one line.

I get that and I having nothing against that code here.  (Sorry again to intrude.)

I just comment because, for someone that teaches scientists how to write readable code, the original was very clear.

I always tell my students: one-liners are charming, but they that will bite you after a week or so.